packs/lacework/dashboards/lacework-anomaly-detections.json

```diff
@@ -0,0 +1,197 @@
+{
+  "name": "Lacework Anomaly Detections",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
+  "pages": [
+    {
+      "name": "Lacework Security Summary",
+      "description": null,
+      "widgets": [
+        {
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "height": 2,
+            "width": 5
+          },
+          "title": "",
+          "rawConfiguration": {
+            "text": "\n![Add Images](https://mma.prnewswire.com/media/692641/LaceworkTM_logo_horizontal_Logo.jpg?p=publish)"
```

Comment:
Is the image at this URL expected to be available forever? @jpvajda see here for another example of using images

Reply:
Tracking down our official assets now, will update with something or host it ourselves!